docs/Project-Configuration/Github-Integrations.md

```diff
@@ -62,6 +62,13 @@ evergreen patch
 
 If your project is configured for manual testing, then Evergreen will only add GitHub checks to the PR when prompted, as opposed to for every commit. Commenting `evergreen patch` will trigger this.
 
+
+```
+evergreen patch --alias <alias>
+```
+
+Override the default GitHub PR patch definition with a custom alias.
```

Comment:
We should consider the concern mentioned in [DEVPROD-7617](https://jira.mongodb.org/browse/DEVPROD-7617) for this feature as well.

Reply:
If someone creates an alias that is a clone of the GithubPRAlias but with two tasks less and replaces GithubPRAlias with that one using `evergreen patch --alias <alias>`, I don't think there is anything preventing that.

Reply:
That's true, but you have to be a project admin or go through code review in order to add a new alias.

Reply:
True, but not to use it. And there may already exist an alias with less tasks created for a different purpose.

Reply:
That's true, but users of `evergreen patch` have to explicitly opt in to manual patching with a comment. I think it's okay to ship this feature and then add additional protections later if we need to explicitly prevent users from working around branch protections.

Reply:
I'm okay with adding additional protections later if we create a ticket for it now.

Reply:
I think since this does mention it'll override that could be okay; having discussed this more maybe it is the better user experience for us to just allow users to override their own definitions (for any actually _required_ checks they should be using the merge queue anyway).

Reply:
That would be a better user experience. It would probably be a good idea to let some users (who are known to use branch protections to determine what can be added to the queue) know that this is the case so that they know that they need to set it up in a way that is more specific (on specific tasks rather than just a variant or overall evergreen status) so that it can't be bypassed in this way if they don't want that to be possible. I'll add a comment to the ticket to focus on user communication and documentation instead of fixing it. But if you disagree and think we should keep everything as is, please feel free to close it.

Reply:
@malikchaya2 

> I'm okay with adding additional protections later if we create a ticket for it now.

Opened https://jira.mongodb.org/browse/DEVPROD-8066 to track.

Reply:
thank you!